utils/utils.go

```diff
@@ -60,3 +65,21 @@ func (utils *Utils) ParseEventXML(eventXML string) EtwEvent {
 
 	return EtwEvent{Provider: provider, EventID: eventID, TimeCreated: timeCreated, EventRecordID: eventRecordID}
 }
+
+func (utils *Utils) GetEventRecordIdBookmark(bookmarkStore store.PersistentKeyValueStoreInterface, query string) int {
+	stringEventRecordId, err := bookmarkStore.GetValue(query)
+	if (err != nil) {
+		utils.logger.Println("Unable to get event record ID bookmark for query:", query)
```

Comment:
Make this log explain there's an error here

Reply:
Done.